nemo_text_processing/text_normalization/rw/taggers/time.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+# Copyright (c) 2024, DIGITAL UMUGANDA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nemo_text_processing.text_normalization.en.graph_utils import GraphFst
+import pynini
+from pynini.lib import pynutil
+
+
+class TimeFst(GraphFst):
+    def __init__(self):
+        super().__init__(name="time", kind="classify")
+        
+        hours = pynini.string_map([
```

Comment:
Mind moving this and the minutes into a data file? (It's easier to debug and maintain if we just keep it as text and read from there.)